extensions/amp-script/0.1/amp-script.js

```diff
@@ -133,6 +136,25 @@ export class AmpScript extends AMP.BaseElement {
     });
   }
 
+  /**
+   * @override
+   */
+  onMeasureChanged() {
+    super.onMeasureChanged();
```

Comment:
No need to call this, the default `onMeasureChanged` doesn't do anything.

Reply:
Done